builtin/types/plasma_cash/plasma_cash.proto

```diff
@@ -0,0 +1,79 @@
+syntax = "proto3";
+
+import "github.com/gogo/protobuf/gogoproto/gogo.proto";
+import "github.com/loomnetwork/go-loom/types/types.proto";
+
+
+message PlasmaBlock {
+    BigUInt uid = 1;
+    repeated PlasmaTx transactions = 2; // List of transactions included in Block
+    bytes signature = 3; // Signature on the block’s hash
+    bytes merkle_hash = 4; // The block's merkle root from its included transactions
+    bytes hash = 5; // The hash of the RLP encoded unsigned block’s bytes.
+}
+
+message PlasmaTx { 
+    uint64 uid = 1; // The slot of the UTXO - Currently uint64, subject to change.
+    BigUInt previous_block = 2 [(gogoproto.customname)="PreviousBlock"]; //Each time a transaction is created, it MUST refer to a previous block which also included that transaction. A transaction is considered a “deposit transaction”, if it’s the first UTXO after a user deposits their coin in the Plasma Chain. This transaction mints coins from nowhere in the Plasma Chain and as a result its previous block is 0.
+    BigUInt denomination = 3; // How many coins are included in that UTXO. Currently this is always 1 since we’re using ERC721 tokens which are unique, however in future iterations this can be any number.
+    Address new_owner = 4 [(gogoproto.customname)="NewOwner"]; //  The new owner of the transaction.
+    bytes signature = 5; // Signature of the block’s hash
+    bytes hash = 6;   // The hash of the RLP encoded unsigned transaction’s bytes. If the transaction is a deposit transaction (its prevblock is 0), its hash is the hash of its uid
+    bytes merkle_hash = 7; // The hash of the RLP encoded signed transaction’s bytes
+    Address sender = 8; // The transaction’s sender, derived from the hash and the signature
+}
+
+
+message ListTransactionsRequest {
+    BigUInt block_height = 1; 
+}
+
+message ListTransactionsResponse {
+    repeated PlasmaTx transactions = 1;
+}
+
+// This only originates from the validator
+message SubmitBlockToMainnetRequest {
+    uint64 slot = 1;
+    BigUInt deposit_block_number = 2  [(gogoproto.customname)="DepositBlockNumber"];
+    uint64 denomination = 3;
+    Address address = 4;
+}
```

Comment:
When you submit a block to mainnet you just submit its merkle hash. 
Request just contains a bytes32 hash (i.e. `plasma.submitBlock(hash)`)

Reply:
this request just tells the validator to start the submission process, so really just need a block height, maybe just remove rest of parameters